catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -421,6 +421,10 @@ private String createCenterPoint(List<Serializable> values) {
       centerPoint = geometries.get(0).getCentroid();
     }
 
+    if (centerPoint == null || centerPoint.isEmpty()) {
```

Comment:
❓ This now means `null` will be added to the Solr document at line 310. What is the impact?
❓ Shouldn't we also annotate the method with `@Nullable`?

Reply:
Since we already return `null` when there's not geometry, I'd say that it's fine. I've also seen no issues when manually testing.

I'd rather not add `@Nullable` to this method because it's private. I know adding it to private methods is a gray area right now, and my personal preference is to not add it.